Online/State/RealizedPlayerState.cs

```diff
@@ -110,6 +130,20 @@ public override void ReadTo(OnlineEntity onlineEntity)
                 //if (pl.slugOnBack != null)
                 //    pl.slugOnBack.slugcat = (slugOnBack?.FindEntity() as OnlinePhysicalObject)?.apo?.realizedObject as Player;
 
+
+                if ((pl.graphicsModule as PlayerGraphics).tail != null)
+                {
+                    (pl.graphicsModule as PlayerGraphics).tail[3].pos = tailPos3;
+                    (pl.graphicsModule as PlayerGraphics).tail[3].lastPos = tailPos3;
```

Comment:
typically setting lastpos causes animation to look more jittery than it needs to. lastpos is used to interpolate the draw-position of stuff since the game runs physics at 40fps but graphics at 60. forcing lastpos effectively causes stuff to move at 40fps while the rest of things animate at 60